package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "eslint": "^7.27.0",
     "eslint-plugin-jest": "^24.3.6",
+    "eslint-plugin-node": "^11.1.0",
```

Comment:
Time to explain setting up a linter!

There's actually very little that goes on. The linter is an external tool, not part of our code. Simply install `eslint` as a `devDependency` (`npm i -D eslint`) and you've now got it installed. The `npm run lint` (and `npm run fix` that you built) are nice scripts to have around.

As you get more and more code, it can be useful to get `eslint` plugins for specific packages you use. Since we're on NodeJS, it's nice to have `eslint-plugin-node`, which just adds rules to catch common mistakes and bad practices in Node code. Since we have `jest` for tests, `eslint-plugin-jest` does the same, tells us when we're doing things not recommended for Jest users.